app/controllers/branches_controller.rb

```diff
@@ -53,10 +53,10 @@ def destroy
       @branch = Branch.find(@leaf.branch_id)
       if @branch.leafs.count <= 1
         if @branch.destroy
-          flash[:success] = "Branch pruned!"
+          flash[:success] = 'Branch pruned!'
         end
       elsif @leaf.destroy
-          flash[:success] = "Leaf pruned!"
+        flash[:success] = 'Leaf pruned!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.